src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
You should link the compiled CSS file, not the SCSS file. Please change 'styles/main.scss' to 'styles/main.css'.